site/content/en/docs/Third Party Content/examples.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Third Party Examples"
+linkTitle: "Examples"
+date: 2021-09-09
+description: "Community contributed Dedicated Game Server examples on Agones."
+weight: 20
+---
+
+## Minetest
+
+[Minetest Game Server for Agones](https://github.com/paulhkim80/agones-example-minetest) - [Minetest](https://www.minetest.net/) example
```

Comment:
The way this renders (https://8762e22-dot-preview-dot-agones-images.appspot.com/site/docs/third-party-content/examples/) doesn't make it visually obvious that there are two links (I didn't realize it until I looked at the source).

I think it would be really useful if you left the `## Minetest` heading and then wrote a short paragraph below describing a) what Minetest is, and b) How the example integrates it with agones. Basically, what you put into the [README](https://github.com/googleforgames/agones/blob/5252a8d3055ba323a0e1e88c08acac9ef830028e/examples/minetest/README.md) in your other PR. 

WDYT?

Reply:
Made the edit as suggested.

I'll later edit the sample repo readme.md to have more info and differ from what has been described here.